intermediate-week-1/controllers/auth/signup.js

```diff
@@ -12,9 +14,17 @@ const signup = async (req, res, next) => {
    *
    */
 
-  res.status(201).json({
-    message: 'Stored user details successfully',
-  });
+  try {
+    const { firstName, lastName, email, password, passwordConfirm } = req.body;
+    const user = new User({ firstName, lastName, email, password, passwordConfirm });
+    await user.save();
+
+    res.status(201).json({
+      message: 'Stored user details successfully',
+    });
+  } catch (err) {
+    res.status(501).send('Could not store user');
```

Comment:
Nice error handling as well. 

In practice its good for all success responses and error responses to follow the same format. 

Thats why I recomment checking out the material on error handling.